cdap-data-fabric/src/main/java/co/cask/cdap/data2/metrics/HBaseDatasetMetricsReporter.java

```diff
@@ -106,24 +106,26 @@ private void reportHBaseStats() throws IOException, DatasetManagementException {
     }
   }
 
-  private void report(Map<TableId, HBaseTableUtil.TableStats> tableStats) {
+  private void report(Map<TableId, HBaseTableUtil.TableStats> tableStats) throws IOException {
+    Map<String, String> reverseNamespaceMap = hBaseTableUtil.getHBaseToCDAPNamespaceMap();
```

Comment:
I am not sure. @gsps1 Any thoughts? Do you know if we have unit tests for these metrics?